pkg/controller/sharedinfo/shared_info.go

```diff
@@ -40,33 +40,35 @@ type SharedInfo struct {
 	NsxNcpDeploymentSpec  *unstructured.Unstructured
 }
 
+func getAdaptorName() (string, error) {
+	osReleasePath, envSet := os.LookupEnv("OS_RELEASE_PATH")
+	if !envSet {
+		osReleasePath = "/host/etc/os-release"
```

Comment:
OsReleaseFile in https://github.com/vmware/nsx-container-plugin-operator/blob/52d9f351cc7812a7c3878833453640638a478df9/pkg/types/names.go#LL41C2-L41C15

Reply:
Thanks, done.